### PR TITLE
Add Multinomial Distribution and Modify Examples

### PR DESCRIPTION
**Pull request by matteolepur**
This will fail because of a runtime error caused by the IntSliceSampler receiving invalid parameters (0,0) in the DiscreteUnif generator. Can anyone help me with this?

### CODE REVIEW

File: src/main/java/blang/distributions/Multinomial.bl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	laws{
+		
+		logf(numberofTrials) { return logFactorial(numberofTrials) }
+		
+		logf(numberofSuccesses, probabilities, numberofTrials) {
+			var sum0 = 0.0
+			for (int i: 0..< numberofSuccesses.size()) {
+				if (probabilities.get(i) < 0.0 || probabilities.get(i) > 1.0) return NEGATIVE_INFINITY
+				if (numberofSuccesses.get(i) < 0.0) return NEGATIVE_INFINITY
+				if (numberofTrials <= 0 || numberofSuccesses.get(i) < numberofTrials) return NEGATIVE_INFINITY
```
Comment (alexandrebouchard) [inline src/main/java/blang/distributions/Multinomial.bl]: A first problem: ``numberofSuccesses.get(i) < numberofTrials`` should be ``numberofSuccesses.get(i) > numberofTrials``

File: src/main/java/blang/distributions/Multinomial.bl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+				sum0 += (numberofSuccesses.get(i) * log(probabilities.get(i)))
```
Comment (alexandrebouchard) [inline src/main/java/blang/distributions/Multinomial.bl]: Line 26 could cause ``0 * -infty = NaN``, which is not the right behaviour

File: src/main/java/blang/distributions/Multinomial.bl [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+			return sum0
+			}
+		}
+		
+		logf(numberofSuccesses, probabilities, numberofTrials) {
```
Comment (alexandrebouchard) [inline src/main/java/blang/distributions/Multinomial.bl]: This second block should be combined with the first since it uses the same arguments.

File: src/test/java/blang/Examples.xtend [hunk 4]
```diff
@@ -77,6 +78,18 @@ class Examples {
   
   public val List<Instance<? extends Model>> all = new ArrayList
   
+<<<<<<< HEAD
+    public val multinomial = add(
```
Comment (alexandrebouchard) [inline src/test/java/blang/Examples.xtend]: Add test cases with probabilities equal to zero and one
